### PR TITLE
ZH-trans: add 2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+date: 2016-02-24
+slug: kubecon-eu-2016-kubernetes-community-in
+url: /blog/2016/02/Kubecon-Eu-2016-Kubernetes-Community-In
+---
+-->
+
+<!--
+KubeCon EU 2016 is the inaugural [European Kubernetes](http://kubernetes.io/) community conference that follows on the American launch in November 2015. KubeCon is fully dedicated to education and community engagement for[Kubernetes](http://kubernetes.io/) enthusiasts, production users and the surrounding ecosystem.
+-->
+KubeCon EU 2016 是首届[欧洲 Kubernetes](http://kubernetes.io/) 社区会议，随后于2015年11月在美国推出。KubeCon 致力于为 [Kubernetes](http://kubernetes.io/) 爱好者、产品用户和周围的生态系统提供教育和社区参与。
```
Comment (chenrui333) [inline content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md]:
```suggestion
KubeCon EU 2016 是首届[欧洲 Kubernetes](http://kubernetes.io/) 社区会议，紧随 2015 年 11 月召开的北美会议。KubeCon 致力于为 [Kubernetes](http://kubernetes.io/) 爱好者、产品用户和周围的生态系统提供教育和社区参与。
```

File: content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+<!--
+Come join us in London and hang out with hundreds from the Kubernetes community and experience a wide variety of deep technical expert talks and use cases.
+-->
+快来加入我们在伦敦，与 Kubernetes 社区的数百人一起出去，体验各种深入的技术专家讲座和用例。
+
+<!--
+Don’t miss these great speaker sessions at the conference:
+-->
+不要错过这些伟大的演讲者会议：
```
Comment (chenrui333) [inline content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md]:
```suggestion
不要错过这些优质的演讲：
```

File: content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* “A Practical Guide to Container Scheduling” by Mandy Waite, Developer Advocate, Google -* [http://sched.co/6BZa](http://sched.co/6BZa)  
+
+* “[Kubernetes in Production in The New York Times newsroom](https://kubeconeurope2016.sched.org/event/67f2/kubernetes-in-production-in-the-new-york-times-newsroom?iframe=yes&w=i:0;&sidebar=yes&bg=no#?iframe=yes&w=i:100;&sidebar=yes&bg=no)” Eric Lewis, Web Developer, New York Times -* [http://sched.co/67f2](http://sched.co/67f2)
+* “[Creating an Advanced Load Balancing Solution for Kubernetes with NGINX](https://kubeconeurope2016.sched.org/event/6Bc9/creating-an-advanced-load-balancing-solution-for-kubernetes-with-nginx?iframe=yes&w=i:0;&sidebar=yes&bg=no#?iframe=yes&w=i:100;&sidebar=yes&bg=no)” by Andrew Hutchings, Technical Product Manager, NGINX -* http://sched.co/6Bc9
+* And many more http://kubeconeurope2016.sched.org/
+-->
+
+* “Kubernetes 硬件攻防：通过旋钮、推杆和滑块探索 Kubernetes API” 作者 Ian Lewis 和 Brian Dorsey，开发者倡导者，谷歌 -* [http://sched.co/6Bl3](http://sched.co/6Bl3)
```
Comment (chenrui333) [inline content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md]: hack 黑客技术，跟 信息安全攻防不太一样。
developer advocate 开发布道师
```suggestion
* “Kubernetes 硬件黑客：通过旋钮、推杆和滑块探索 Kubernetes API” 演讲者 Ian Lewis 和 Brian Dorsey，开发布道师，谷歌 -* [http://sched.co/6Bl3](http://sched.co/6Bl3)  
```

File: content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+* “A Practical Guide to Container Scheduling” by Mandy Waite, Developer Advocate, Google -* [http://sched.co/6BZa](http://sched.co/6BZa)  
+
+* “[Kubernetes in Production in The New York Times newsroom](https://kubeconeurope2016.sched.org/event/67f2/kubernetes-in-production-in-the-new-york-times-newsroom?iframe=yes&w=i:0;&sidebar=yes&bg=no#?iframe=yes&w=i:100;&sidebar=yes&bg=no)” Eric Lewis, Web Developer, New York Times -* [http://sched.co/67f2](http://sched.co/67f2)
+* “[Creating an Advanced Load Balancing Solution for Kubernetes with NGINX](https://kubeconeurope2016.sched.org/event/6Bc9/creating-an-advanced-load-balancing-solution-for-kubernetes-with-nginx?iframe=yes&w=i:0;&sidebar=yes&bg=no#?iframe=yes&w=i:100;&sidebar=yes&bg=no)” by Andrew Hutchings, Technical Product Manager, NGINX -* http://sched.co/6Bc9
+* And many more http://kubeconeurope2016.sched.org/
+-->
+
+* “Kubernetes 硬件攻防：通过旋钮、推杆和滑块探索 Kubernetes API” 作者 Ian Lewis 和 Brian Dorsey，开发者倡导者，谷歌 -* [http://sched.co/6Bl3](http://sched.co/6Bl3)  
+
+* “rktnetes: 容器运行时和 Kubernetes 的新功能” 作者 Jonathan Boulle, CoreOS 的开发人员和团队负责人 -* [http://sched.co/6BY7](http://sched.co/6BY7)
```
Comment (chenrui333) [inline content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md]:
```suggestion
* “rktnetes: 容器运行时和 Kubernetes 的新功能” 演讲者 Jonathan Boulle, CoreOS 的主程 -* [http://sched.co/6BY7](http://sched.co/6BY7)
```

File: content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+[在这里](https://ti.to/kubecon/kubecon-eu-2016)获取您的 KubeCon EU 门票。
+
+<!--
+Venue Location: CodeNode * 10 South Pl, London, United Kingdom  
+Accommodations: [hotels](https://skillsmatter.com/contact-us#hotels)   
+Website: [kubecon.io](https://www.kubecon.io/)   
+Twitter: [@KubeConio](https://twitter.com/kubeconio) #KubeCon
+Google is a proud Diamond sponsor of KubeCon EU 2016. Come to London next month, March 10th & 11th, and visit booth #13 to learn all about Kubernetes, Google Container Engine (GKE) and Google Cloud Platform!  
+-->
+地点：英国伦敦，南 Pl， * 10
```
Comment (chenrui333) [inline content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md]:
```suggestion
会场地址：CodeNode * 英国伦敦南广场 10 号
```

File: content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+酒店住宿：（酒店）(https://skillsmatter.com/contact-us)
```
Comment (chenrui333) [inline content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md]:
```suggestion
酒店住宿：[酒店](https://skillsmatter.com/contact-us)
```

File: content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+网站：[kubecon.io] (https://www.kubecon.io/)
+推特：[@KubeConio] (https://twitter.com/kubeconio)
+谷歌是 KubeCon EU 2016 的钻石赞助商。下个月3月10-11号来伦敦，参观13号展位，了解 Kubernetes，Google Container Engine（GKE），Google Cloud Platform 的所有信息!
```
Comment (chenrui333) [inline content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md]:
```suggestion
谷歌是 KubeCon EU 2016 的钻石赞助商。下个月 3 月 10 - 11 号来伦敦，参观 13 号展位，了解 Kubernetes，Google Container Engine（GKE），Google Cloud Platform 的所有信息!
```

File: content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+<!--
+_KubeCon is organized by KubeAcademy, LLC, a community-driven group of developers focused on the education of developers and the promotion of Kubernetes._  
+
+-* Sarah Novotny, Kubernetes Community Manager, Google  
+-->
+
+_KubeCon 是由 KubeAcademy、LLC 组织的，这是一个由社区驱动的开发者团体，专注于开发人员的教育和 kubernet.com 的推广
+-* Sarah Novotny, Kubernetes 社区经理，谷歌
```
Comment (chenrui333) [inline content/zh/blog/_posts/2016-02-00-Kubecon-Eu-2016-Kubernetes-Community-In.md]:
```suggestion
-* Sarah Novotny, 谷歌的 Kubernetes 社区经理
```

**Comment by xichengliudui** [comment]
已更新。

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @xichengliudui, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/52) has no assignee. will not be reported to the task issues

**Pull request merged**